razzle.config.js

```diff
@@ -54,6 +57,23 @@ module.exports = {
       };
     }
 
+    if (target === 'web') {
```

Comment:
Service worker setup is failing for me on local dev. Recommend to enable just for prod build.

Reply:
Done